src/demo/views/CustomWorkspace.tsx

```diff
@@ -12,38 +12,39 @@ const customWorkspaces: CustomWorkspaces = {
       },
       panels: {
         spectraPanel: { display: true, open: true },
-        informationPanel: { display: false, open: false },
+        informationPanel: { display: true, open: false },
         integralsPanel: { display: false, open: false },
         rangesPanel: { display: false, open: false },
         structuresPanel: { display: false, open: false },
         filtersPanel: { display: false, open: false },
         zonesPanel: { display: false, open: false },
-        automaticAssignmentPanel: { display: false, open: false },
+        automaticAssignmentPanel: { display: true, open: false },
         databasePanel: { display: false, open: false },
         multipleSpectraAnalysisPanel: { display: true, open: true },
         peaksPanel: { display: false, open: false },
         predictionPanel: { display: false, open: false },
         summaryPanel: { display: false, open: false },
+        matrixGenerationPanel: { display: true },
       },
       toolBarButtons: {
         baselineCorrection: false,
-        exclusionZones: false,
-        exportAs: false,
+        exclusionZones: true,
+        exportAs: true,
         fastFourierTransform: false,
         import: true,
         integral: false,
```

Comment:
you can remove it because it is by default hidden and the same for the panels where it the `display` value is false

Reply:
In fact I just copied the preferences from the config panel and pasted them without any changes. You think we should not copy the values that are false ? This could be a problem if the default value changes in the future.

Reply:
you can keep them like that or remove the default ones, in any way, I have a file for the default workspace values where I merge it with the values that come from you or the predefined workspace to be sure that all the objects are listed there.